src/libs/antares/study/parts/short-term-storage/STStorageOutputCaptions.cpp

```diff
@@ -0,0 +1,36 @@
+
+#include <vector>
+#include "STStorageOutputCaptions.h"
+
+namespace Antares::Data::ShortTermStorage
+{
+const std::vector<std::string> groups = {
+    "PSP_open", 
+    "PSP_closed", 
+    "Pondage", 
+    "Battery", 
+    "Other1", 
+    "Other2", 
+    "Other3", 
+    "Other4", 
+    "Other5"};
+
+const std::vector<std::string> variables = {"injection", "withdrawal", "level"};
```

Comment:
Make these structures `static`.

Or possibly re-use structures defined in properties.h/properties.cpp